app/lib/course_subjects.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_relative "../../app/lib/degree_subjects"
```

Comment:
This is to stop the noisy warnings discussed here: https://ukgovernmentdfe.slack.com/archives/C03SR5B5EGH/p1666705025952029